git-user-count/user-count.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```

Comment:
This is slightly more portable (and is a best practice):

```suggestion
#!/usr/bin/env bash
```

Reply:
Thanks.